src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,80 @@
+package chess.domain.board;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import chess.domain.piece.Color;
+import chess.domain.piece.Piece;
+import chess.domain.position.Direction;
+import chess.domain.position.Position;
+
+public class Board {
+
+    private static final String NOT_EXIST_PIECE = "[ERROR] 입력한 위치에 말이 존재하지 않습니다.";
+    private static final String CAN_NOT_MOVE_PIECE = "[ERROR] 해당 위치는 말이 움직일 수 없습니다.";
+    private static final String CAN_NOT_PLACE_PIECE = "[ERROR] 해당 위치로 말이 도달할 수 없습니다.";
+    private static final String CAN_NOT_CATCH_PIECE = "[ERROR] 잡을 수 없는 말 입니다.";
+
+    private final Map<Position, Piece> value;
+
+    public Board() {
+        this.value = new HashMap<>();
+    }
+
+    public Board(final Map<Position, Piece> value) {
+        this.value = new HashMap<>(value);
+    }
```

Comment:
생성자 하나를 주 생성자로 만들어주세요.
```suggestion
    // 부 생성자
    public Board() {
        this(new HashMap<>());
    }

    // 주 생성자
    public Board(final Map<Position, Piece> value) {
        this.value = new HashMap<>(value);
    }
```